client/ayon_core/tools/publisher/control.py

```diff
@@ -711,23 +850,32 @@ def add_error(self, plugin, error, instance):
 
         # Make sure the cached report is cleared
         plugin_id = self._plugins_proxy.get_plugin_id(plugin)
-        if not error.title:
-            if hasattr(plugin, "label") and plugin.label:
-                plugin_label = plugin.label
-            else:
-                plugin_label = plugin.__name__
-            error.title = plugin_label
+        self._ensure_validation_title(plugin, error)
```

Comment:
to be even more clear about what it's doing, could we call it
`_set_error_title_if_missing`?

Reply:
I noticed that – if I'm not mistaken – I could restore PublishValidationErrors to its initial state. Since no modifications were still needed, removing this method, which didn't exist originally, restored the class completely.

Reply:
I'm okay with that :)